dtfs-central-api/src/services/state-machines/utilisation-report/event-handlers/manually-set-incomplete/manually-set-incomplete.event-handler.test.ts

```diff
@@ -1,14 +1,70 @@
-import { UtilisationReportEntityMockBuilder } from '@ukef/dtfs2-common';
-import { handleUtilisationReportManuallySetIncompleteEvent } from './manually-set-incomplete.event-handler';
-import { NotImplementedError } from '../../../../../errors';
+import { EntityManager } from 'typeorm';
+import {
+  AzureFileInfoEntity,
+  DbRequestSource,
+  MOCK_AZURE_FILE_INFO,
+  UTILISATION_REPORT_RECONCILIATION_STATUS,
+  UtilisationReportEntity,
+  UtilisationReportEntityMockBuilder,
+  getDbAuditUpdatedByUserId,
+} from '@ukef/dtfs2-common';
+import { handleUtilisationReportManuallySetIncompleteEvent } from '.';
 
 describe('handleUtilisationReportManuallySetIncompleteEvent', () => {
-  // TODO FN-1862 - update tests when functionality implemented
-  it('throws a NotImplementedError', () => {
+  const requestSource: DbRequestSource = {
+    platform: 'TFM',
+    userId: 'abc123',
+  };
+  const updatedByUserId = getDbAuditUpdatedByUserId(requestSource);
+
+  const mockSave = jest.fn();
+
+  const mockRepository = {
+    save: mockSave,
+  };
+
+  it(`calls the correct repo methods and sets the report to '${UTILISATION_REPORT_RECONCILIATION_STATUS.PENDING_RECONCILIATION}' when the report was previously uploaded`, async () => {
```

Comment:
It is generally best to avoid words like 'correct' in test names and instead explicitly spell out what we mean by correct because if someone is not familiar with the context 'correct' doesn't really tell them anything. I think here for example `sets the report to '${UTILISATION_REPORT_RECONCILIATION_STATUS.PENDING_RECONCILIATION}' when the report was previously uploaded and saves using the entity manager` would perhaps be a better name.